build.sbt

```diff
@@ -67,6 +68,7 @@ lazy val `core-common` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     name := "otel4s-core-common",
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-core" % CatsVersion,
+      "org.typelevel" %%% "vault" % VaultVersion,
```

Comment:
This has been stable for a long time, but it's not strictly necessary.  It turns out to be a very good match for the Otel spec.